src/createEpicMiddleware.js

```diff
@@ -45,6 +48,7 @@ export function createEpicMiddleware(epic, options = defaultOptions) {
           return output$;
         })
         ::switchMap(output$ => options.adapter.output(output$))
+        ::mergeMap(action => of(action, asap))
```

Comment:
hmm I'm pretty reluctant to schedule every action asap. Seems like it could create hard to understand behavior for people, as well as more complex stack traces--separately, I think `::observeOn(asap)` would be the more idiomatic way of achieving asap scheduling, though they both would have the same effect seemingly. Don't worry about changing it just yet, need to figure out out if this is the best solution in general.